app/src/main/java/com/vadim/playlistmaker/TrackApiService.kt

```diff
@@ -0,0 +1,10 @@
+package com.vadim.playlistmaker
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface TrackApiService {
+    @GET("search")
```

Comment:
Чтобы искать только треки, тут необходимо добавить query параметр:
```suggestion
    @GET("search?entity=song")
```